lib/reqTemplate.js

```diff
@@ -138,7 +140,7 @@ function splitAndPrepareTAsseblyTemplate(templateSpec, options) {
 
                 // TODO: Enable single brace structures outside of URI once
                 // the specs are migrated to double brace.
-                if (!inDoubleBrace && options.isURI) {
+                if (!inDoubleBrace) {
                     if (/^\+/.test(currentTemplate)) {
```

Comment:
hm.. We don't really evaluate the templates over public uri specs now, but I'm ok with leaving support.